tests/test_nanopub.py

```diff
@@ -30,6 +30,29 @@ def test_construction_with_bnode_introduced_concept(self):
         )
         assert str(publication.introduces_concept) == test_concept_uri
 
+    def test_construction_with_derived_from_as_list(self):
```

Comment:
Could you rename this test module to test_publication.py? (I fucked that up)

Reply:
Done :)